package-lock.json

```diff
@@ -1,12056 +1,8 @@
 {
   "name": "@yext/answers-core",
   "version": "1.3.2",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
I think we want to keep lockfileVersion 2

Reply:
Node 8 just came out and I think we are good to use that one as well since it's very similar to 7 except it drops support for some of the older node versions. I'm not sure how we'd setup npm 7 and node 14, however we can easily setup npm 8 and node 14 with: `nvm use 14 && nvm install-latest-npm`